Dockerfile

```diff
@@ -20,13 +20,13 @@ COPY controllers/ controllers/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
-  GO111MODULE=on go build -ldflags="-w -s" -a -o /usr/bin/controller
```

Comment:
My mistake. This should be `controller` everywhere. Can you update it?

Reply:
Of course, done.